packages/jsii-pacmak/lib/targets/version-utils.ts

```diff
@@ -55,10 +67,93 @@ export function toPythonVersionRange(semverRange: string): string {
     .join(', ');
 }
 
+/**
+ * Converts an original version number from the NPM convention to the target
+ * language's convention for expressing the same. For versions that do not
+ * include a prerelease identifier, this always returns the assembly version
+ * unmodified.
+ *
+ * @param assemblyVersion the assembly version being released
+ * @param target          the target language for which the version is destined
+ *
+ * @returns the version that should be serialized
+ */
+export function toReleaseVersion(
```

Comment:
We need this entire pre-release support in pacmak and the python quirks documented somewhere.

I see this package doesn't have a proper README.
Can we start by adding this?

Reply:
As a side note, I recommend adding something similar to 'prlinter' for jsii, so that all `feat` require a README update. Pretty useful IMO.

Reply:
> As a side note, I recommend adding something similar to 'prlinter' that we have in the cdk for the jsii repo. Enforce that all feat require a README update. Pretty useful IMO.

I have mixed feelings on this based on how often I have needed to apply labels to bypass this validation in my CDK contributions (for good reasons) in the past. But you're probably right, still... At least it ensures people have fought about it.

Anyhow - the fact there is no README.md entry yet is because this is a draft and I didn't want to have to come back to it multiple times if the implementation changes dramatically as I get into testing it properly.

Reply:
I understand it's annoying to add the label, but the value outweighs it, IMO.